src/components/Clients/DeleteClientModal.jsx

```diff
@@ -0,0 +1,109 @@
+// React Imports
+import React, { useContext } from 'react';
+// Material UI Imports
+import Button from '@mui/material/Button';
+import ClearIcon from '@mui/icons-material/Clear';
+import CheckIcon from '@mui/icons-material/Check';
+import Dialog from '@mui/material/Dialog';
+import DialogActions from '@mui/material/DialogActions';
+import DialogContent from '@mui/material/DialogContent';
+import DialogContentText from '@mui/material/DialogContentText';
+// Utility Imports
+import { runNotification } from '../../utils';
+// Custom Hook Imports
+import { useStatusNotification } from '../../hooks';
+// Context Imports
+import { UserListContext } from '../../contexts';
+// Component Imports
+import FormSection from '../Form/FormSection';
+
+/**
+ * DeleteClientModal Component - Component that allows users to delete other user's
+ * Pod URLs from a user's list stored on their own Pod
+ *
+ * @memberof Clients
+ * @name DeleteClientModal
+ */
+
+const DeleteClientModal = ({
+  showDeleteClientModal,
+  setShowDeleteClientModal,
+  selectedClientToDelete
+}) => {
+  const { state, dispatch } = useStatusNotification();
+  const { removeUser } = useContext(UserListContext);
+
+  // Event handler for deleting client from client list
+  const handleDeleteClient = async (event) => {
+    event.preventDefault();
+    runNotification(
+      `Deleting "${selectedClientToDelete?.person}" from client list...`,
+      5,
+      state,
+      dispatch
+    );
+    try {
+      await removeUser(selectedClientToDelete);
+    } finally {
+      runNotification(
+        `"${selectedClientToDelete?.person}" deleted from client list...`,
+        5,
+        state,
+        dispatch
+      );
+      setTimeout(() => {
+        setShowDeleteClientModal(false);
+      }, 2000);
+    }
+  };
+
+  return (
+    <Dialog
+      open={showDeleteClientModal}
+      aria-labelledby="dialog-title"
+      aria-describedby="dialog-description"
+      onClose={() => setShowDeleteClientModal(false)}
+    >
+      <FormSection
```

Comment:
Followup: This shouldn't really be a form. I get it's because you want to use the notifications system, but that seems to me like a sign that we need a better notifications system. We should be able to send notifications from any kind of component, not just forms. After we rework notifications to be robust, we should consider reworking this form too. That's for the future though.

Reply:
Fully agreed. I thought we already had an issue for that, but I guess not. I will create one.

Revamping the UI/UX of `<StatusNotification>` is definitely out of the scope of this PR.  I just made the code here the same as it was when the notifications were showing in the client table (and it's how they are coded in the `AddClientModal`, and all the other forms we still have - so it's easier for whichever dev wants to tackle that issue.

Reply:
Yup I totally agree that what you did here was the right thing given the current state of the code base. Mostly just wanted to call it out, since we don't want this design pattern to proliferate too much. Nice work either way